proposals/spiffe.adoc

```diff
@@ -0,0 +1,140 @@
+
+1. **SPIFFE ID**: A specification defining how workloads identify themselves to each other; such an ID is implemented as a Uniform Resource Identifier (URI).
+1. **SPIFFE Verifiable Identity Document (SVID)**: a specification for encoding SPIFFE IDs in a cryptographically-verifiable document.
+1. **SPIFFE Workload API**: An API specification to issue and/or retrieve SVIDs.
+
+The SPIFFE Workload API does not require a calling workload to 1) have a priori knowledge of its identity; or 2) possess authentication token(s) when calling the API.
+
+Implementations of the SPIFFE Workload API can 1) run on and across multiple platforms; and 2) identify running workloads at a process “and” kernel level, making it suitable for use with container schedulers like Kubernetes.
+
+Building upon work done at Bell Labs (Plan 9), Google (LOAS), and others, **SPIRE (aka the “SPIFFE Runtime Environment”)** is an open-source software implementation of SPIFFE that can bootstrap and issue cryptographically verifiable identity to workloads running on heterogeneous environments and organizational boundaries. SPIRE consists of two (2) components:
+
+1. **SPIRE Server**: provides a central registry of SPIFFE IDs, and the attestation policies describing which workloads can assume said identities. Attestation policies describe the properties a workload must exhibit to be assigned a SPIFFE ID, and are described as a mix of process attributes (such as a Linux UID, or Kubernetes service account) and infrastructure attributes (such as running in a Amazon EC2 instance with a particular tag).
+
+1. **SPIRE Agent**: runs on any kernel and exposes the local workload API to any process that needs a SPIFFE ID, key, and/or trust bundle. On *nix systems, this API is exposed locally through a Unix domain socket. By verifying the attributes of a calling workload, the workload API avoids requiring the workload to supply a secret in order to authenticate.
+
+SPIRE’s 12-month roadmap is exciting and will deliver multiple features:
+
+* Production readiness, including HA mode, versioned APIs, documented SLOs, >80% test coverage, and functional testing in release train.
+* Support for automatic bootstrapping and node attestation on public cloud platforms (Amazon Web Services, Microsoft Azure, and Google Cloud Platform).
+* Support for automatic bootstrapping and node attestation on virtualization platforms (VMWare and OpenStack).
+* Support for Microsoft Windows-based workloads.
+* SPIFFE Workload API client libraries in Go, C, Java, and Javascript, with support for TLS negotiation and JWT signing.
+* gRPC support for the SPIFFE Workload API.
+* SPIFFE Workload API certificate helpers for Linux and Windows.
+* A standards conformance test suite.
+* Secure introduction to popular products, including Lyft Envoy and Hashicorp Vault.
+
+*Sponsor / Advisor from TOC*: Brian Grant <briangrant@google.com>
+
+*Preferred maturity level*: Inception
+
+*Unique Identifier*: spiffe
+
+*License*: ALv2
```

Comment:
Should this make it clear is this apache?

Reply:
`Apache-2.0` is the SPDX notation.

https://spdx.org/licenses/Apache-2.0.html